modules/gapi/include/opencv2/gapi/streaming/streaming.hpp

```diff
@@ -0,0 +1,25 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2020 Intel Corporation
+
+
+#ifndef OPENCV_GAPI_STREAMING_API_HPP
+#define OPENCV_GAPI_STREAMING_API_HPP
+
+#include <opencv2/gapi/gkernel.hpp> // GKernelPackage
+#include <opencv2/gapi/own/exports.hpp> // GAPI_EXPORTS
+
+namespace cv {
+namespace gapi {
+namespace streaming {
+
+GAPI_EXPORTS_W cv::gapi::GKernelPackage kernels();
+
+} // namespace streaming
+} // namespace gapi
+} // namespace cv
+
+
+#endif // OPENCV_GAPI_STREAMING_API_HPP
```

Comment:
This header makes no sense.

Also, I don't see a reason to export this `kernels()` function - it must be always included into the _default_ package.

Reply:
Removed the header. Added `kernels()` to `streaming/format.hpp`. It was already in the default kernels package. Not sure if it's good now

Reply:
It is ok that is is already in the default kernel package.